src/DetailView/EventTable.js

```diff
@@ -14,6 +17,13 @@ class EventTable extends Component {
 							<th key={key} 
 								className={css(AppStyles.tableCell, AppStyles.tableHeaderCell)}>
 								{attribute.Name}
+								<a className="showOnHover"
+									href={Utils.getLink(
+										"analytics?eventTypeId=" + this.props.currentEventTypeId + "&entityId="
+										+ this.props.entityId + "&attributeName=" + attribute.Name
+										+ '&types=["attribute-distribution"]')}>
+									<InsertChartButton style={{height: "16px", width: "16px"}}/>
```

Comment:
I cannot do anything about the naming of [this icon](https://material.io/icons/#ic_insert_chart) in Material UI ;)